main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
+    noun = input("Choose a noun: ")
```

Comment:
Is there a way to make sure the type of user input is a string? Perhaps we can validate the type (e.g., type = string)?